client/spec/components/ProjectList.spec.jsx

```diff
@@ -1,23 +1,28 @@
 import projectListFactory from "inject!ProjectList";
 
 describe( "ProjectList", () => {
-	let component, dependencies, selectStub;
+	let component, dependencies, selectStub, ProjectList;
 
 	beforeEach( () => {
 		dependencies = {};
-		const ProjectList = projectListFactory( dependencies );
+		ProjectList = projectListFactory( dependencies );
 		selectStub = sinon.stub();
-		component = ReactUtils.renderIntoDocument( <ProjectList projects={ [] } onSelectProject={ selectStub } /> );
 	} );
 
+	function render( props = {} ) {
```

Comment:
It'd be good to update the yeoman generator to account for some of these subtitles.. and also fix the export issue we keep running into.